### DISCUSSION TITLE
Issues with button configs on xbox one controller

### DISCUSSION THREAD

**Discussion by wtfbbqhax** [darthcloud/BlueRetro#479] [General]
Hey all, I've been feeling like there is some issues with device N mappings, but I also could be just misunderstanding the actual implementation. 

Firstly, I've mapped all devices to assume nintendo switch pro controllers, going to n64, and trying to remap the first paired device to be a xbox one controller, it just doesn't change the setup. I still have the X, Y mapped all wrong (xbox controller with switch pro mapping). 

I am honestly very confused about device mappings in general, device 1 for me is _not functional_, i assume it's literally identifying my PC from config, not the first paired controller (more documentation would be nice here). Are "devices" the paired controllers attached in order? or are they "all controllers since device flashed"? Can I reset all devices? Can I see which device is was ladt paired to which N number?

**Comment by darthcloud** [comment]
Is your xbox controller updated to latest FW?

Update via the Xbox accessories Win10 app.

In general things work fine by default without touching config if you got latest FW.

**Reply by wtfbbqhax** [reply]
Ah, good idea. This controller has never been updated since I purchased it. I will report back the results once I test.
